app/views/capacity.html

```diff
@@ -40,6 +40,71 @@
 
 <div id="plotly-div-line" role="presentation"></div>
 
+{% if organisationLevel !== 'offender-manager' %}
+  <h3 class="heading-medium">Capacity breakdown</h3>
+  <table id="example" class="data-table dataTable">
+    <thead>
+      <tr>
+        <th colspan="2" id="team">{{ childOrganisationLevelDisplayText }}</th>
```

Comment:
`id="team"` here I'm guessing will only be true at an LDU level. Perhaps something like `id="child"` would be more appropriate?

Reply:
Copy pasta error I believe, fixed